client/src/components/ProfilePicture/ProfilePicture.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./ProfilePicture.css";
+import { Link } from "react-router-dom";
+
+// Function based React Component
+const ProfilePicture = props => {
+  console.log(props);
+  // Default Class to apply to Component
+  let classList = `ProfilePicture`;
+  let types = ["extra-small", "small", "medium", "large"];
+  if (types.includes(props.type)) {
+    classList += ` profile-pic-${props.type}`;
+  }
```

Comment:
Can you make this more succinct? One option is to set a default value for the `type` property. Then you can use a single string interpolation statement.